client/src/components/Home.js

```diff
@@ -4,22 +4,27 @@ import Button from '@material-ui/core/Button';
 import Icon from '@material-ui/core/Icon';
 import Grid from '@material-ui/core/Grid';
 import { withStyles } from '@material-ui/core/styles';
+import Leaderboard from './Leaderboard';
+import { getTopFive } from '../utils/leaderboad-api';
 
 const styles = theme => ({
   button: { margin: theme.spacing.unit },
   rightIcon: { marginLeft: theme.spacing.unit },
 });
 
 const Home = ({ classes }) => (
-  <Grid container direction="row" justify="center" alignItems="center">
-    <Button className={classes.button} variant="contained" color="primary" size="large">
-      Play
-      <Icon className={classes.rightIcon}>send</Icon>
-    </Button>
-    <Button className={classes.button} variant="contained" color="secondary" size="large">
-      Find a Friend
-    </Button>
-  </Grid>
+  <React.Fragment>
+    <Grid container direction="row" justify="center" alignItems="center">
+      <Button className={classes.button} variant="contained" color="primary" size="large">
+        Play
+        <Icon className={classes.rightIcon}>send</Icon>
+      </Button>
+      <Button className={classes.button} variant="contained" color="secondary" size="large">
+        Find a Friend
+      </Button>
+    </Grid>
+    <Leaderboard getTopFive={getTopFive} />
```

Comment:
Also home component doesn't know about how leaderboard is getting the data, so I think its better that instead of passing as the prop you called getTopFive from `Leaderboard` component itself.

Reply:
I did this so that I could test the component better. In the tests, I can pass mock functions as the props.

Reply:
Thoughts? @nimish-gupta

Reply:
You can mock also mock them in LeaderBoard.test file like `jest.mock(location_of_the_function)`.